graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/types/generateInterface.kt

```diff
@@ -58,7 +57,7 @@ internal fun generateInterface(generator: SchemaGenerator, kClass: KClass<*>): G
         .forEach { builder.field(generateFunction(generator, it, kClass.getSimpleName(), null, abstract = true)) }
 
     generator.classScanner.getSubTypesOf(kClass)
-        .filter { it.isGraphQLIgnored().not() }
+        .filter { generator.config.hooks.isValidAdditionalType(it, inputType = false) }
```

Comment:
this seems weird calling it from inteface generator

Reply:
It is another location in where we add to the `additionalTypes` so I don't see why we should exclude it. 

Also we need that to work for “input only” types as types could be added from interfaces and the implementations for input only but the interface is ignored